doc/examples/plot_seam_carving.py

```diff
@@ -0,0 +1,38 @@
+"""
+============
+Seam Carving
+============
+
+This example demonstrates how images can be resized using seam carving [1]_.
+Resizing often distorts contents in the image. Seam carving tries to resize
+images while trying to keep important content intact. In this example we are
+using the Sobel filter to signify the importance of each pixel.
+
+.. [1] Shai Avidan and Ariel Shamir
+       "Seam Carving for Content-Aware Image Resizing"
+       http://www.cs.jhu.edu/~misha/ReadingSeminar/Papers/Avidan07.pdf
+
+"""
+from skimage import io, data
+from skimage import transform
+from skimage import color, filters
+from matplotlib import pyplot as plt
+
+
+img = data.coins()
+out  = transform.seam_carve(img, 'vertical', 80, energy_func = filters.sobel)
```

Comment:
And when we remove enough seams ( let's say `~100`,) the intial energy map will be pretty irrelevant for the new image